src/rf/apps/core/context_processors.py

```diff
@@ -23,11 +23,11 @@ def page_context(request):
 
 
 def get_client_settings():
-    conn = boto.connect_s3()
-    aws_key = conn.aws_access_key_id
+    creds = provider.Provider('aws')
     client_settings = {
         'signerUrl': reverse('sign_request'),
-        'awsKey': aws_key,
+        'awsKey': creds.access_key,
+        'awsToken': creds.security_token,
```

Comment:
Is this the secret token? It seems like the wrong idea to pass this to the front end.